Client/Frontend/Browser/FindInPageBar.swift

```diff
@@ -191,6 +181,17 @@ class FindInPageBar: UIView {
     static var retrieveSavedText: String? {
         return UserDefaults.standard.object(forKey: FindInPageBar.savedTextKey) as? String
     }
+
+    // MARK: - Theme Applicable
+    func applyTheme(theme: Theme) {
+        let colors = theme.colors
+        topBorder.backgroundColor = colors.borderPrimary
+        searchText.textColor = colors.layerRatingF
```

Comment:
As Crystal commented, could we make this textPrimary? 🙏 
```suggestion
        searchText.textColor = colors.textPrimary
```